libnetdata/config/appconfig.h

```diff
@@ -98,6 +98,36 @@
 #define CONFIG_MAX_NAME 1024
 #define CONFIG_MAX_VALUE 2048
 
```

Comment:
These are also changed that were merged days ago.

Reply:
The same. Conflicts were resolved.

Reply:
Github continued showing me the files, I pressed "ctrl+shift+R" and now everything is ok.